api/client/cluster/client.go

```diff
@@ -160,76 +161,6 @@ func (c *clusterClient) SetLoggingURL(loggingURL string) error {
 
 }
 
-func (c *clusterClient) SetManagementURL(managementURL string) error {
```

Comment:
Why is this removed for a docs PR? If this needs to be removed, please create a new PR just for this change.

Reply:
None of these features have been used as far as i know but i can put them back and file a seperate pr for htose